src/services/codefixes/fixClassDoesntImplementInheritedAbstractMember.ts

```diff
@@ -14,18 +14,22 @@ namespace ts.codefix {
             return changes.length === 0 ? undefined : [{ description: getLocaleSpecificMessage(Diagnostics.Implement_inherited_abstract_class), changes, fixId }];
         },
         fixIds: [fixId],

```

Comment:
I'm probably missing something obvious, but I would have guessed that the solution was to give all the fixes for the same derived-base pair the same fixId.

Reply:
When we get a combined code fix, we want to get the fix for all classes in the file. The fixId is used only for asking for combined code fixes.

Reply:
Makes sense.  Thanks!